chall08/szavarce.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
```

Comment:
This could have crashed if I had a `python2` interpreter in my `PATH` variable before my `python3` if you ever used only-python3 features like `f-strings` and such. I think it would be better to specify `#!/usr/bin/env python3`, just because we asked for `Python3.x`